org.lflang/src/org/lflang/generator/ReactorInstance.java

```diff
@@ -1114,4 +1114,53 @@ private void setInitialWidth() {
      * Cached reaction graph containing reactions that form a causality loop.
      */
     private ReactionInstanceGraph cachedReactionLoopGraph = null;
+
+    private TimeValue let = null;
+    
+    public TimeValue getLogicalExecutionTime() {
```

Comment:
This is confusing to me. Why do you calculate LET values for reactors?

Reply:
Sorry, this is my mistake forgot to delete this part.
I deleted it in https://github.com/lf-lang/lingua-franca/pull/1292/commits/ed0e00e44a54d108d45a0b269986c36333481ba2.